### PR TITLE
Bee/Ike's Loadout System and Better-looking Berets

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
**PORT OF BeeStation/BeeStation-Hornet#645**
(full credit to Ike709)

Adds a slightly modified and adapted version of Ike709's loadout system from Bee. Lots of items have been added on to Bee's original system, so it should be decent enough in terms of variation. Another change is that items actually show on the loadout screen when possible, rather than on Bee where you can't see them until you're already in-game.

Also resprites the departmental berets to be more inclusive and consistent with palettes and sprite design. Because why not. ~~If someone asks me to atomize I will lose it~~

image
/\ don't do this /\

## Why It's Good For The Game
Adds something to spend [CURRENCY NAME HERE] on. Also a highly requested feature from... well, bee again.

## Changelog
:cl:
add: A new loadout system has been added. Finally, something to spend [currency name] on!
add: A few new sub-departments now have berets in respective wardrobes and lockers.
imageadd: Berets have gotten a resprite, and a few more departments have been given their own versions of them.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
atomize

**Comment by triplezeta** [comment]
also NOOOOOO MY CORPORATE SECURITY BERETS

**Comment by MarkSuckerberg** [comment]
fuggem, they didn't match with any of the sec uniforms so they got removed

**Comment by triplezeta** [comment]
but i need my pure black outfit 
image

**Comment by Victor239** [comment]
Bro undelete beret_corporate_officer and beret_corporate_warden right now >:(

**Comment by MarkSuckerberg** [comment]
edgelord moth

File: waspstation/code/modules/clothing/head/berets.dm [hunk 1]
```diff
@@ -77,26 +77,41 @@
...
 /obj/item/clothing/head/beret/sec/navyhos
-	name = "head of security's beret"
-	desc = "A special beret with the Head of Security's insignia emblazoned on it. A symbol of excellence, a badge of courage, a mark of distinction."
+	name = "head of security's navy beret"
+	desc = "A navy-colored beret with the Head of Security's insignia emblazoned on it. A symbol of excellence, a badge of courage, a mark of distinction."
 	icon_state = "hosberet"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 25, "energy" = 35, "bomb" = 25, "bio" = 10, "rad" = 0, "fire" = 50, "acid" = 60)
+
+/obj/item/clothing/head/beret/sec/navyhos/black
+	name = "head of security's bleck beret"
```
Comment (triplezeta) [inline waspstation/code/modules/clothing/head/berets.dm:91]: bleck

**Comment by Superyodeler** [comment]
Just when I had forgotten about the moth hair incident

**Comment by MarkSuckerberg** [comment]
[Tie MetaCoins to a goon-style job cosmetics system.](https://trello.com/c/NaRGNJ9r/53-tie-metacoins-to-a-goon-style-job-cosmetics-system)

**Pull request merged**